expenses/config.py

```diff
@@ -0,0 +1,20 @@
+from google.cloud import storage
+from google.oauth2 import service_account
+import os
+
+google_storage_cred = {
+    "type": "service_account",
+    "project_id": "hackgt-cluster",
+    "private_key_id": os.environ["GOOGLE_PRIVATE_ID"],
+    "private_key": os.environ['GOOGLE_PRIVATE'].replace(r'\n', '\n'),
+    "client_email": "hackgt-storage@hackgt-cluster.iam.gserviceaccount.com",
+    "client_id": "112216606107657666067",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://oauth2.googleapis.com/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/hackgt-storage%40hackgt-cluster.iam.gserviceaccount.com"
+}
```

Comment:
any of these values that are specific to our configuration should be provided via environment variables / secrets

Reply:
i wasnt sure exactly how to use a json file secret, can you help me figure it out?

Reply:
Personally, I think having more environment variables is easier than dealing with adding secrets via a file.  But it is possible.

The two main steps are:

1. Locally, create the secret file and then create a secret `--from-file` as shown here https://kubernetesbyexample.com/secrets/
2. Add to your deployment.yaml file as shown in [this repo](https://github.com/HackGT/reddit-crawler/blob/master/deployment.yaml).  This makes the secret accessible as a file at /etc/reddit-crawler/config.ini

Reply:
Ok great thanks, I got it to work